docs/releases/1.10.txt

```diff
@@ -55,6 +55,31 @@ Minor features
 :mod:`django.contrib.auth`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Support for the Argon2_ password hash has been added.  Argon2 is the winner
+  of the recent `Password Hashing Competition`_, which has been organized
+  by the community in similar style as the NIST competitions that selected
+  the hash SHA-3 and cipher AES.
```

Comment:
I don't completely understand your objection.  You are afraid we oversell argon2?  Would it help if we add a note that it is a young algorithm?

BTW, a lot of people probably wouldn't know about the NIST competitions.  So if you want to leave out AES and SHA-3, then maybe it's better to completely leave out the reference to NIST.